### PR TITLE
Rubocop -a for all Amazon specs

### PR DESCRIPTION
**Pull request by Ladas**
Rubocop -a for all Amazon specs

### CODE REVIEW

File: spec/models/manageiq/providers/amazon/cloud_manager/metrics_capture_spec.rb [hunk 1]
```diff
@@ -23,16 +23,16 @@
       }
       with_aws_stubbed(stubbed_responses) do
         expect(vm.perf_collect_metrics('realtime')).to eq([
-          {"amazon-perf-vm" => described_class::VIM_STYLE_COUNTERS},
```
Comment (durandom) [inline spec/models/manageiq/providers/amazon/cloud_manager/metrics_capture_spec.rb]: if you move the `[` to the next line - does it look nicer?

File: spec/models/manageiq/providers/amazon/cloud_manager/provision_workflow_spec.rb [hunk 2]
```diff
@@ -395,7 +395,8 @@
       stub_get_next_vm_name
 
       # the dialogs populate this
-      values.merge!(:src_vm_id => template.id, :vm_tags => [])
+      values[:src_vm_id] = template.id
```
Comment (durandom) [inline spec/models/manageiq/providers/amazon/cloud_manager/provision_workflow_spec.rb]: that was rubocop?

Comment (Ladas) [inline spec/models/manageiq/providers/amazon/cloud_manager/provision_workflow_spec.rb]: yes

File: spec/models/manageiq/providers/amazon/cloud_manager/metrics_capture_spec.rb (hunk 1 above)
Comment (Ladas) [inline spec/models/manageiq/providers/amazon/cloud_manager/metrics_capture_spec.rb]: can we keep this pure rubocop -a ?

**Pull request merged**